app/assets/javascripts/app/views/publisher/getting_started_view.js

```diff
@@ -44,7 +44,7 @@ app.views.PublisherGettingStarted = Backbone.View.extend({
     }, 1400);
 
     // hide some popovers when a post is created
-    this.$('.button.creation').click(function() {
+    this.$('.button').click(function() {
```

Comment:
This seems to be broken because the share button in the publisher doesn't have a `button` class. Instead this could be
```javascript
this.$("button#submit").click(function() {
```
(I didn't test it yet, so make sure it does the right thing)